### PR TITLE
document that icarus ignores COCOTB_HDL_TIME* variables

### PR DESCRIPTION
**Pull request by Millennium-Software-Design**
this is in response to #1102 , specifically this comment: https://github.com/cocotb/cocotb/issues/1102#issuecomment-576011634

### CODE REVIEW

File: documentation/source/simulator_support.rst [hunk 1]
```diff
@@ -43,6 +43,11 @@ to the top component as shown in the example below:
     `endif
     endmodule
 
+Time unit and precision
+~~~~~~~~~~~~~~~~~~~~~~~
+
+Setting the time unit and time precision is not possible from the command-line, and therefore make variables ``COCOTB_HDL_TIMEUNIT`` and ``COCOTB_HDL_TIMEPRECISION`` are ignored.
```
Comment (cmarqu) [inline documentation/source/simulator_support.rst]:
```suggestion
Setting the time unit and time precision is not possible from the command-line,
and therefore the make variables :make:var:`COCOTB_HDL_TIMEUNIT` and :make:var:`COCOTB_HDL_TIMEPRECISION` are ignored.
```

**Comment by cmarqu** [comment]
Thanks for the PR, just one small semantic markup thing.

File: documentation/source/simulator_support.rst [hunk 2]
```diff
@@ -43,6 +43,12 @@ to the top component as shown in the example below:
     `endif
     endmodule
 
+Time unit and precision
+~~~~~~~~~~~~~~~~~~~~~~~
+
+Setting the time unit and time precision is not possible from the command-line,
+and therefore make variables :make:var:``COCOTB_HDL_TIMEUNIT`` and :make:var:``COCOTB_HDL_TIMEPRECISION`` are ignored.
```
Comment (cmarqu) [inline documentation/source/simulator_support.rst]:
```suggestion
and therefore the make variables :make:var:`COCOTB_HDL_TIMEUNIT` and :make:var:`COCOTB_HDL_TIMEPRECISION` are ignored.
```
This syntax requires single backticks only.

Comment (Millennium-Software-Design) [reply documentation/source/simulator_support.rst]: oops!  Sorry, will fix right away

**Comment by cmarqu** [comment]
Thanks, and congratulations to your first commit!

**Pull request merged**